docs/clipboad.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get/Set Clipboard
+---
+
+## Get Clipboard
+
+For iOS 13+ real devices, Apple security preferences require the WebDriverAgentRunner application to be in foreground in order to be able to receive the system clipboard content.
+Otherwise an empty string is always returned.
+Consider using [Activate App](execute-methods.md/#mobile-activateapp) and [Background App](execute-methods.md/#mobile-backgroundapp) to change the foreground application.
```

Comment:
I assume it would be useful to add a code example here about how to activate WDA, fetch clipboard and then reactive the app under test